### ISSUE TITLE
RoBERTa in Google Colab

### ISSUE THREAD

**Issue by pat266** [amaiya/ktrain#387]
Hello,

I am currently End-to-End Question-Answering in Google Colab and I would like to change the model from BERT to RoBERTa. From what I have read from the FAQ section, I need to download the necessary files and then load the model? What should I do now?

My current code is like this (very similar to https://nbviewer.jupyter.org/github/amaiya/ktrain/blob/master/examples/text/question_answering_with_bert.ipynb):
```
import ktrain
from ktrain import text

from pathlib import Path
import shutil

INDEXDIR = '/tmp/myindex' # a temporary directory, not important
dirpath = Path(INDEXDIR) # 'dataset'

# check to see if it exists, if it is, then delete it
if dirpath.exists() and dirpath.is_dir():
    shutil.rmtree(dirpath)

text.SimpleQA.initialize_index(INDEXDIR)
text.SimpleQA.index_from_list(clean_data, INDEXDIR, commit_every=len(clean_data),
                              multisegment=True, procs=2, # these args speed up indexing, can use !cat /proc/cpuinfo in Google Colab to view CPU's spec
                               breakup_docs=True,         # this slows indexing but speeds up answer retrieval
                              references=data_label, min_words=7)
```

Thank you

**Comment by amaiya** [comment]
You can supply it to [SimpleQA](https://amaiya.github.io/ktrain/text/qa/core.html#ktrain.text.qa.core.SimpleQA) using the parameter `bert_squad_model`.   `SimpleQA` simply takes the value of this parameter and supplies it to `TFAutoModelForQuestionAnswering.from_pretrained`.  This assumes the model you're supplying is a TensorFlow model.  If the model you're planning to use only exists as a Pytorch model, then this code may need to be changed to support loading PyTorch checkpoints as TensorFlow models (which is very easy to do).  

Is the model you're using a PyTorch model?

**Comment by pat266** [comment]
Yes, I am planning to use 'deepset/roberta-base-squad2' (https://huggingface.co/deepset/roberta-base-squad2) as the 'bert_squad_model', but it is PyTorch only model, so how can the code be changed?

Also, I am planning to use 'roberta-large' as the 'bert_emb_model.' In order to use it as TensorFlows model, I only need to add TF in front, so it will become 'TFroberta-large' correct?

**Comment by amaiya** [comment]
This should now be fixed in v0.27.3 - thank you.
